src/cares_wrap.cc

```diff
@@ -1064,30 +1064,29 @@ int ParseSoaReply(Environment* env,
   /* Can't use ares_parse_soa_reply() here which can only parse single record */
   unsigned int ancount = cares_get_16bit(buf + 6);
   unsigned char* ptr = buf + NS_HFIXEDSZ;
-  char* name;
-  char* rr_name;
+  int rr_type, rr_len;
```

Comment:
Should this line be here?

Reply:
It does seem to be added by accident?

Reply:
@cjihrig @BridgeAR Do you mean line# 1067?

Reply:
Yes. It looks related to merge conflicts or something (I'm pretty sure I removed that line recently).

Reply:
@cjihrig Yes you are right, that line is not in the mainline. Is it okay if I update the PR with removal of that line (rr_len and rr_type)?